pkg/cli/admin/inspect/discovery.go

```diff
@@ -0,0 +1,56 @@
+package inspect
+
+import (
+	"context"
+	"fmt"
+	"k8s.io/apimachinery/pkg/util/json"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+
+	"k8s.io/client-go/discovery"
+)
+
+func InspectDiscovery(ctx context.Context, destDir string, discoveryClient discovery.CachedDiscoveryInterface) error {
+	if err := writeDiscovery(ctx, filepath.Join(destDir, "discovery-apis.yaml"), discoveryClient, "/apis"); err != nil {
```

Comment:
should we name the file as `aggregated-discovery-apis.yaml` to highlight it contains the aggregated discovery data?

Reply:
> should we name the file as `aggregated-discovery-apis.yaml` to highlight it contains the aggregated discovery data?

Up to @ardaguclu

Reply:
There is no other discovery anymore but `aggregated-discovery-apis.yaml` seems to be better reflect the feature that apis are retrieved from.